syntax.php

```diff
@@ -83,6 +83,11 @@ function handle($match, $state, $pos, Doku_Handler $handler){
         $urlparam = array();
         foreach($paramm as $key => $value) {
             switch($key) {
+                case 'list':
+                    if(preg_match('/(\w|-)+/',$value)) { 
```

Comment:
the regexp isn't anchored it would still match anything as long as it contains at least one letter. try ``/^[-\w]+$/``